src/Text.mo

```diff
@@ -55,7 +92,7 @@ module {
     return x
   };
 
-  /// Returns the concatenation of `t1` and `t2`, `t1 # t2`.
+  /// Returns `t1 # t2`, where `#` is the `Text` concatenation operator.
```

Comment:
So I think these trivial functions are here to be passed in as higher order functions, but

maybe it's a good place to put examples using the operators. I've heard anecdotally that it's hard to find some of these text operators

Reply:
Awesome. I think it might be good to do these for the trivial comparison functions too.